### DISCUSSION TITLE
How to delete a date field from a github roadmap?

### DISCUSSION THREAD

**Discussion by romw314** [community/community#61766] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to?
How to delete a date field from a GitHub Project v2 - roadmap view?

**Comment by Beargruug** [comment]
Here's how you can do it:

1. Go to the GitHub repository that contains the project board you want to edit.

2. Click on the "Projects" tab at the top of the repository.

3. Open the project board that contains the date field you want to delete.

4. Switch to the "Roadmap" view. This view shows all the issues and pull requests with their associated dates.

5. Find the card with the date field you want to delete.

6. Click on the card to open it.

7. Look for the date field in the card's details. Depending on how the date field was added, it might be in a custom field or in the card's description.

8. To remove the date field, simply clear the date value. If the date field is in the card's description, edit the description and remove the date information.

9. Save the changes to the card.

The date field will no longer be displayed in the roadmap view as it has no value.

I hope this will work for you!

**Reply by wallace-zero1-systems** [reply]
Its worked for me! thanks a lot!

**Comment by romw314** [comment]
But I have an iteration in the view and I want to delete it.
Can I delete it without deleting the view/project?

**Comment by Beargruug** [comment]
Unfortunately, GitHub doesn't have a built-in feature to delete just one iteration while keeping everything else intact.

However, there are a couple of workarounds you can try:

1. **Editing the Iteration Date**: You can change the start and end dates of the iteration to match those of another iteration or just make them generic so that they don't represent any specific timeframe. This way, the iteration will still technically be there, but it won't have any significant impact on your roadmap view.
2. **Removing the Iteration Tag**: If you've been using labels or tags to define your iterations, you can remove the iteration tag from the relevant issues or pull requests. This will unlink those items from the specific iteration, and they won't show up under it in the roadmap view anymore.
3. **Using a Custom Field**: If your project supports custom fields, you could create a custom field specifically for iterations. Then, you can manually update or clear the custom field value for the items you want to remove from that particular iteration.

I hope this helps somehow!

**Comment by romw314** [comment]
But it always show the iterations in my roadmap. I want to delete it from the entrie project. I want to delete all of the iterations.

**Reply by srendonv** [reply, marked as answer]
Suppose you want to delete an iteration field from your project. You have to go to the project's setting; on the left panel, you will find all the custom fields. Then click on the delete icon to the custom field you remove

**Reply by romw314** [reply]
Thanks, **it worked**!
